tvm/python/tvm/topi/adreno/pooling.py

```diff
@@ -62,6 +62,7 @@ def _schedule(PaddedInput, Pool):
             s[OL].vectorize(s[OL].op.axis[-1])
         else:
             s[Pool].compute_at(s[Out], tx)
+            s[Pool].vectorize(s[Pool].op.axis[-1])
```

Comment:
Thank you. LGTM, but I'd like to suggest small refactoring. After line 54 we could add `OL = Pool`. And after that we can remove this if condition and keep only true branch.